src/MicrosoftTeams.schema.json

```diff
@@ -147,11 +147,12 @@
                     },
                     "scopes": {
                         "type": "array",
-                        "description": "Specifies whether the tab offers an experience in the context of a channel in a team, or an experience scoped to an individual user alone. These options are non-exclusive. Currently, configurable tabs are only supported in the teams scope.",
-                        "maxItems": 1,
+                        "description": "Specifies whether the tab offers an experience in the context of a channel in a team, in a 1:1 or group chat, or in an experience scoped to an individual user alone. These options are non-exclusive. Currently, configurable tabs are only supported in the teams and groupchats scopes.",
+                        "maxItems": 2,
                         "items": {
                             "enum": [
-                                "team"
+                                "team",
+                                "groupchat"
```

Comment:
What about staticTabs? We should probably support groupchat there as well.

Reply:
We're not planning to support it for the moment, but I'll talk to PM and let him figure out.